packages/data-helper/src/hooks/useSchemaFields/useSchemaFields.gql-queries.js

```diff
@@ -0,0 +1,15 @@
+import gql from 'graphql-tag';
+
+export const SCHEMA_FIELDS_QUERY = gql`query schemaFields($type: String!) {
+    __type(name: $type) {
```

Comment:
I think it's still better to get the whole schema. It can be used for other types than publication.

Reply:
Ok, I'll get the whole schema then